base_tier_validation/models/tier_definition.py

```diff
@@ -58,6 +58,14 @@ def _get_tier_validation_model_names(self):
         default=lambda self: self.env["res.company"]._company_default_get(
             "tier.definition"),
     )
+    has_comment = fields.Boolean(
+        string='Comment',
+        default=False,
+    )
+    approve_level = fields.Boolean(
```

Comment:
Does meaning of `approve_level` means to approve by `sequence` field? If so, i would change the name to,
```
approve_sequence = fields.Boolean(
    string='Approve by sequence',
    default=False,
    help="Approval order by the specified sequence number",
```

Reply:
Edited